torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -974,6 +987,33 @@ llvm::Error PyTorchModelLoader::loadMin(const torch::jit::Node *ptNode) {
   return addGlowNodeValue(outputs[0], output);
 }
 
+llvm::Error PyTorchModelLoader::loadPRelu(const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  RETURN_IF_ERR(checkInputAndOutputSizes(inputs, 2, outputs, 1));
+
+  glow::NodeValue in;
+  ASSIGN_VALUE_OR_RETURN_ERR(in, getGlowNodeValue(inputs[PReluInputs::input]));
+  glow::NodeValue weight;
+  ASSIGN_VALUE_OR_RETURN_ERR(weight,
+                             getGlowNodeValue(inputs[PReluInputs::weight]));
+
+  // Do broadcasting.
+  auto targetDim = in.dims();
+  auto weightDim = weight.dims();
+  RETURN_ERR_IF_NOT(
+      weightDim.size() == targetDim.size() || weightDim.size() == 1,
+      glow::strFormat(
+          "Weigh dimensions must be 1, or the number of channels, got %lu",
```

Comment:
"weight"
Also if weightDim is equal to targetDim we don't need a broadcast here.